custom-connectors/HR Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,962 @@
+                      },
+                      "xDescription": {
+                        "type": "string",
+                        "description": "xDescription"
+                      },
+                      "xPositionCode": {
+                        "type": "string",
+                        "description": "xPositionCode"
+                      },
+                      "xPositionTitle": {
+                        "type": "string",
+                        "description": "xPositionTitle"
+                      },
+                      "xRecordStatus": {
+                        "type": "string",
+                        "description": "xRecordStatus"
+                      }
+                    },
+                    "description": "xPositionLookup"
+                  },
+                  "xRecordStatus": {
+                    "type": "string",
+                    "description": "xRecordStatus"
+                  },
+                  "xSalary": {
+                    "type": "object",
+                    "properties": {
+                      "Amount": {
+                        "type": "string",
+                        "description": "Amount"
+                      },
+                      "Currency": {
+                        "type": "string",
+                        "description": "Currency"
+                      }
+                    },
+                    "description": "xSalary"
+                  },
+                  "xSeparationDate": {
+                    "type": "string",
+                    "description": "xSeparationDate"
+                  },
+                  "xSeparationReason": {
+                    "type": "string",
+                    "description": "xSeparationReason"
+                  },
+                  "xStartDate": {
+                    "type": "string",
+                    "description": "xStartDate"
+                  },
+                  "xState": {
+                    "type": "string",
+                    "description": "xState"
+                  },
+                  "xUpdatedOn": {
+                    "type": "string",
+                    "description": "xUpdatedOn"
+                  },
+                  "xVeteranStatus": {
+                    "type": "string",
+                    "description": "xVeteranStatus"
+                  },
+                  "xWorkableId": {
+                    "type": "string",
+                    "description": "xWorkableId"
+                  },
+                  "xWorkPhone": {
+                    "type": "string",
+                    "description": "xWorkPhone"
+                  },
+                  "xZipCode": {
+                    "type": "string",
+                    "description": "xZipCode"
+                  }
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "GetEmployees",
```

Comment:
@HRCIntegrationsTeam Summary values are visible to public, So please make them free from grammatical mistakes.

Documentation - [Summary](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the Summary values in the swagger